provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -235,14 +242,22 @@ data:
         "aci-vrf": {{config.aci_config.vrf.name|json}},
         "default-endpoint-group": {
             "policy-space": {{config.kube_config.default_endpoint_group.tenant|json}},
+        {% if config.flavor == "eks" %}
```

Comment:
do this and some of the other changes below also apply to the localhost flavor?

Reply:
As of now, no.